test/Integration/SqlOutputBindingIntegrationTests.cs

```diff
@@ -123,6 +123,8 @@ public void AddProductArrayTest(SupportedLanguages lang)
 
         [Theory]
         [SqlInlineData()]
+        [UnsupportedLanguages(SupportedLanguages.JavaScript)]
+        // Collectors are CSharp specific and are not relevant for JavaScript
```

Comment:
Put these kinds of comments on the same line as the `UnsupportedLanguages` attribute so it's clear that's what they're tied to.

I'd also suggest making this comment more generic so we don't have to update it for future languages - `Collectors are only available in C#`